pkg/gui/gui.go

```diff
@@ -778,3 +784,18 @@ func (gui *Gui) OnUIThread(f func() error) {
 		return f()
 	})
 }
+
+func (gui *Gui) UpdateWindowTitle() error {
+	if runtime.GOOS != "windows" {
+		return nil
+	}
+	path, getWdErr := os.Getwd()
+	if getWdErr != nil {
+		return getWdErr
+	}
+	cmd := exec.Command("cmd", "/C", "title", filepath.Base(path))
```

Comment:
I wonder if we should be saying 'RepoName - Lazygit' or 'Lazygit - RepoName' to make it clearer. Happy for you to make the call on this.

Also we'll need to ensure that if our repo has spaces in the name that this command still works as expected.

Reply:
Another thing: we have an existing approach for running something in a shell that's platform-agnostic. That is, we could go:
```
func (c *OSCommand) SetWindowTitle(title string) error {
	return c.Cmd.NewShell("title " + c.Quote(title)).Run()
}
```
But I'm not on windows and don't know whether that works off-hand

Reply:
> I wonder if we should be saying 'RepoName - Lazygit' or 'Lazygit - RepoName' to make it clearer. Happy for you to make the call on this.
> 
> Also we'll need to ensure that if our repo has spaces in the name that this command still works as expected.

From the other windows I have open it seems a bit mixed, but Google Chrome and VSCode both have the "[custom title] - [application name]" format so I think maybe that makes most sense.

Reply:
> Another thing: we have an existing approach for running something in a shell that's platform-agnostic. That is, we could go:
> 
> ```
> func (c *OSCommand) SetWindowTitle(title string) error {
> 	return c.Cmd.NewShell("title " + c.Quote(title)).Run()
> }
> ```
> 
> But I'm not on windows and don't know whether that works off-hand

Makes sense, but how would I invoke it from [gui.loadNewRepo](https://github.com/jesseduffield/lazygit/blob/master/pkg/gui/gui.go#L677)?

Reply:
Nevermind, I got it :)

> Also we'll need to ensure that if our repo has spaces in the name that this command still works as expected.

Good thinking. Turns out that we need to write out the title without quotation marks, even when there are spaces.

So the command is like this: `title This is my window title`.

![image](https://user-images.githubusercontent.com/5260237/150121856-1c275cbb-1f4b-4a93-88b5-008e3b47f498.png)